bloodview/src/graph.c

```diff
@@ -38,10 +38,13 @@
 #define GRAPH_EXCESS 1024
```

Comment:
Typo in the commit message? Should be "amplitude" I assume?

Reply:
Yep! Fixed. :smile_cat: